### PR TITLE
docs: add ma-hidden documentation

### PR DESCRIPTION
**Pull request by carlotamolintr**
Hello all, 

I just did the ma-hidden documentation. 

Do we need to add examples too?

I wait for your comments

### CODE REVIEW

File: docs/components/layout/MaHidden.docs.mdx [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+- **below**: Hides the content below the desired breakpoint. Whose values can be "desktop", "tablet" and "null.
+- **above**: Hides the content above the desired breakpoint. Whose values can be "mobile" , "tablet" and "null".
+
+There is another property that sets if content wrapper should be an inline element:
+
+- **inline:** This is a boolean property. Set as true, the html tag will be a `<div>` Set as false will be as `<span>`.
+
+## When to use it
+
+In that cases that we need to show or hide an element in a specific breakpoints
```
Comment (edufarre) [inline docs/components/layout/MaHidden.docs.mdx:20]:
```suggestion
In those cases that we need to show or hide an element in specific breakpoints
```

**Review by edufarre — APPROVED** [review]
I believe the component is quite explanatory by itself, I don't think having examples would bring more value in this case 

🚀

**Review by alexcastelo95 — APPROVED** [review]
Looks good. Is it possible that we need to add an I-Frame to take us to Figma as well?

If not it can be done in another iteration.

**Comment by alexcastelo95** [comment]
> Looks good. Is it possible that we need to add an I-Frame to take us to Figma as well?
> 
> If not it can be done in another iteration.

Come to think of it, maybe the MaHidden is difficult to explain in Figma. 😂

**Pull request merged**